tutorials/docker-compose-on-container-optimized-os.md

```diff
@@ -0,0 +1,148 @@
+OS](/container-optimized-os/) stable image.
+
+1.  Open the [Google Cloud Platform console](https://console.cloud.google.com).
+1.  [Create a new Compute Engine instance](https://console.cloud.google.com/compute/instancesAdd).
+1.  Select the desired **Zone**, such as "us-central1-f".
+1.  Select the desired **Machine type**, such as "micro" (f1-micro).
+1.  Change the **Boot disk** to "Container-Optimized OS stable".
+1.  Check the box to allow HTTP traffic in the **Firewall** section.
+1.  Click the **Create** button to create the Compute Engine instance.
+
+## Getting the sample code
+
+1.  After the instance is created, click the **SSH** button to open a terminal
+    connected to the machine.
+1.  Download the [Hello World
+    sample](https://github.com/docker/dockercloud-hello-world) using the `git`
+    command.
+
+        git clone https://github.com/docker/dockercloud-hello-world.git
+        cd dockercloud-hello-world
+
+## Running Docker Compose
+
+[Container-Optimized OS](/container-optimized-os/) has many
+[benefits](/container-optimized-os/docs/concepts/features-and-benefits), but
+since it is [locked down by
+default](/container-optimized-os/docs/concepts/security), it is difficult to
+run software that is not bundled in a container. For example, the general
+instructions for installing [Docker Compose](https://docs.docker.com/compose/)
+will not work because very few parts of the filesystem are mounted as
+executable. Instead, you can run [Docker Compose
+image](https://hub.docker.com/r/docker/compose/).
+
+1.  Download and run the Docker Compose image. Check the [tags for Docker
+    Compose](https://hub.docker.com/r/docker/compose/tags/) to use the latest
+    version.
+
+        docker run docker/compose:1.13.0 version
+
+1.  Run the Docker Compose command to run the sample code.
+
+    So that the Docker Compose container has access to the Docker daemon, mount
+    the Docker socket with the `-v /var/run/docker.sock:/var/run/docker.sock`
+    option.
+
+    To make the current directory available to the container, use the `-v
+    "$(pwd):$(pwd)"` option to mount it as a volume and the `-w="$(pwd)"` to
+    change the working directory.
+
+        docker run -it \
+            -v /var/run/docker.sock:/var/run/docker.sock \
+            -v "$(pwd):$(pwd)" \
+            -w="$(pwd)" \
+            docker/compose:1.13.0 up
+
+1.  With the `docker run` command still running, open the [Google Cloud
+    Platform Console instances
+    page](https://console.cloud.google.com/compute/instances). Click the link
+    to your instance's **External IP** address.
+
+    You should see a "Hello World" message appear.
+
+1.  With the SSH window open, press Control-C on your keyboard to stop the
+    sample application.
+
+## Making an alias to Docker Compose
+
+The command
+
+    docker run -it \
+        -v /var/run/docker.sock:/var/run/docker.sock \
+        -v "$(pwd):$(pwd)" \
+        -w="$(pwd)" \
+        docker/compose:1.13.0 up
```

Comment:
Except they don't tag a `:latest` for this image. I tried running just `docker/compose` and it failed for that reason.